client/user/src/mealorder/meal-order-client-app.html

```diff
@@ -19,9 +19,8 @@
   -->
 <link rel="import" href="shell/meal-order-app-namespace.html">
```

Comment:
URLs should be relative. First is block with dependencies imports, then after empty line goes block with project imports.